flow-typed/layers.js

```diff
@@ -0,0 +1,189 @@
+//arc-layer
+declare const defaultProps = {
```

Comment:
@IgorWilbert I would assume that:
* You want to call this definition object `ArcLayerPropTypes` or similar, rather than a `const`.
* You would want to declare the ArcLayer constructor as accepting this prop object type definition:  `declare function ArcLayer(props : ArcLayerPropTypes)`;
* Ideally flow allows a prop object type declaration to inherit from another: `declare ArcLayerPropTypes extends LayerPropTypes`

But again I don't know enough about flow type declaration file syntax, perhaps the above is not possible.